tailwind.config.js

```diff
@@ -2,10 +2,22 @@
 export default {
   content: ["./src/**/*.{html,js,svelte,ts}"],
   theme: {
-    color: {
+    colors: {
       black: "#000",
       white: "#fff",
-      green: "#28EB83",
+      green: {
+        light: "#48EE95",
+        DEFAULT: "#28EB83",
+        dark: "#0AD86A",
+      },
+      gray: {
+        DEFAULT: "#F5F5F5",
+        one: "#e5e7eb",
+        two: "#d1d5db",
+        three: "#9ca3af",
+        four: "#6b7280",
+        five: "#4b5563",
```

Comment:
DEFAULT should always be the shade in the middle. Is this the one?
Use the names: lightest, lighter, light, default, dark, darker, and darkest to keep the convention